core/src/main/java/contacts/core/BroadQuery.kt

```diff
@@ -37,22 +37,40 @@ import contacts.core.util.unsafeLazy
  * ## Usage
  *
  * Here is an example query that returns the first 10 [Contact]s, skipping the first 5, where the
- * any Contact data (e.g. name, email, address, phone, etc) matches the search term "john", ordered
- * by the Contact display name primary (given name first) in ascending order (ignoring case).  Only
- * Contacts with at least one RawContact belonging to the given account and groups are included.
- * Only the full name and email address attributes of the [Contact] objects are included.
+ * any Contact data (e.g. name, email, address, phone, note, etc) partially matches the search term
+ * "john", ordered by the Contact display name primary (given name first) in ascending order
+ * (ignoring case). Include only Contacts with at least one RawContact belonging to the given
+ * account and groups. Include only the name and email properties of [Contact]s.
+ *
+ * In Kotlin,
  *
  * ```kotlin
- * import contacts.core.Fields.Name
- * import contacts.core.Fields.Address
- * import contacts.core.ContactsFields.DisplayNamePrimary
+ * val contacts : List<Contact> = broadQuery.
+ *      .accounts(account)
+ *      .groups(groups)
+ *      .include { Name.all + Address.all }
+ *      .whereAnyContactDataPartiallyMatches("john")
+ *      .orderBy(ContactsFields.DisplayNamePrimary.asc())
+ *      .offset(5)
+ *      .limit(10)
+ *      .find()
+ * ```
+ *
+ * In Java,
```

Comment:
I made sure all core API class docs have sample Java code. This became more important because Java users probably can't benefit from the syntactic sugar added in this PR 😞